code/modules/projectiles/guns/energy/nuclear.dm

```diff
@@ -30,7 +30,6 @@
 	name = "miniature energy gun"
 	desc = "A small, pistol-sized energy gun with a built-in flashlight. It has two settings: stun and kill."
 	icon_state = "mini"
-	item_state = "gun"

```

Comment:
Why did you change this?

Reply:
Because I made my own in hand sprites that work much better.

Reply:
This is dealt with by a parent*